File: Modul-2/soal-5/PRAK205_2410817210016_MUHAMMADRIZKYAKBAR.c
#include <stdio.h>
#include <math.h>

int main() {

    float A, B, C, keliling, luas;

    printf("Masukkan Nilai : ");
    scanf("%f %f", &A, &B);

    C = sqrt(B * B - A * A); 

    keliling = A + B + C;

    luas = 0.5 * C * A; 

    printf("Alas = %.0f cm\n", C);
    printf("Tinggi = %.0f cm\n", A);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    return 0;
}

File: Modul-2/soal-5/PRAK205_2410817210016_MUHAMMADRIZKYAKBAR.py
import math

def pythagoras(a, b):
    
    c = math.sqrt(b**2 - a**2)
    
    keliling = a + b + c
    
    luas = (c * a) / 2
    
    return c, a, keliling, luas

nilai = input("Masukkan nilai A dan B : ")
a, b = map(int, nilai.split())

alas, tinggi, keliling, luas = pythagoras(a, b)

print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm^2")
